blocks/columns/columns.js

```diff
@@ -1,3 +1,32 @@
+function areaClick(block) {
+  const pBlock = block.querySelector('div.slim-promo.columns-2-cols p');
+  const pButton = block.querySelector('.slim-promo a.button.secondary');
+  const pH2 = block.querySelector('div.slim-promo.columns-2-cols h2');
+  // toggle the paragraph and the button display
+  if (pBlock.style.display === 'block') {
+    pBlock.style.display = 'none';
+    pButton.style.display = 'block';
+    pH2.style.fontSize = 'initial';
+  } else {
+    pBlock.style.display = 'block';
+    pButton.style.display = 'none';
+    pH2.style.fontSize = '2.4rem'; // enlarge the h2 font size
+  }
+}
+
+function addSlimPromoClick(block, mq) {
+  block.querySelector('div.slim-promo.columns-2-cols > div > div:nth-child(2) > div > a.button.secondary').addEventListener('click', () => {
```

Comment:
That is oddly specific. can it work without `> div > div:nth-child(2) > div >` ?